tensorflow/lite/g3doc/guide/build_rpi.md

```diff
@@ -81,8 +81,9 @@ PATH=../rpi_tools/arm-bcm2708/arm-rpi-4.9.3-linux-gnueabihf/bin:$PATH \
 
 ## Compile natively on Raspberry Pi
 
-The following instructions have been tested on Raspberry Pi Zero, Raspbian
-GNU/Linux 10 (buster), gcc version 8.3.0 (Raspbian 8.3.0-6+rpi1):
+The following instructions have been tested on Raspberry Pi Zero, Raspberry Pi OS 
+(previously called Raspbian) GNU/Linux 10 (Buster), gcc version 8.3.0 
```

Comment:
can remove "(previously called Raspbian)" and use the current name. thanks

Reply:
Do you want this change made throughout the PR? Or just here

Reply:
Throughout, please. Thank you

Reply:
Done.